src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -252,3 +253,28 @@ must be supplied as the value to be matched."
        (sql/do-commands
         (str "ALTER TABLE " table " ALTER COLUMN " column
              " RESTART WITH " restartid))))))
+
+(defn clj->pgobject
```

Comment:
I'm not sure, but I feel like this file and jdbc.clj are becoming mixed up in purpose. There are some functions in jdbc.clj that we draw on for storage purposes, and some in this one as well. I guess in my head I think that this file is about the 'storage' side of the business, versus any query side. But looking at the rest of this file, we've probably lost track of any original intent etc.